### PR TITLE
Bintray release

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: .build-package-script.sh [hunk 1]
```diff
@@ -123,7 +123,7 @@ fi
 
 # Download OpenSSL
 cd $TMP
-wget https://www.openssl.org/source/openssl-$OPENSSL_VERSION.tar.gz -O openssl-$OPENSSL_VERSION.tar.gz
+wget http://www.openssl.org/source/openssl-$OPENSSL_VERSION.tar.gz -O openssl-$OPENSSL_VERSION.tar.gz
```
Comment (thefosk) [inline .build-package-script.sh]: @shashiranjan84 why this change over `http`?

Comment (shashiranjan84) [inline .build-package-script.sh]: i did it because i was getting ssl error. But i did revert it back, may be while rebasing i missed that commit. its good now
